lib/dai-plugin-mcd/contracts/addresses/kovan.json

```diff
@@ -23,7 +23,7 @@
   "MCD_POT": "0xd097295f999608a8a69e892a6b6d51fa0676ca4c",
   "MCD_END": "0x59a779cc66c688f5acf11e7c50fab5558d1d1773",
   "MCD_ESM": "0x90f2b6bb1eb20b6b5ff8a0728b3cb6edb79b225e",
-  "PROXY_ACTIONS": "0x98e3a3d95e380804668c7ad5f851f78c26861f24",
+  "PROXY_ACTIONS": "0xc4ee8c7da07b489d0dd7c729de746935d4fd5891",
```

Comment:
@levity This PR hinges on the inclusion of this update to the 0.2.8 Kovan release. Would the inclusion of this cause issues elsewhere. We could hold off merging this until 0.2.9 alternatively

Reply:
Since #127 is merged in now, this shouldn't cause any problems AFAIK (but @ethanbennett would know for sure)

Reply:
Yeah, this should be fine 👍

Reply:
:+1: